tristan/ding-wall/src/components/AddFooter.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './AddFooter.css';
+
+function AddFooter() {
+  return (
+    <div className="footer">
```

Comment:
No need to wrap the footer in a div, just apply the className to footer directly

- [ ] Remove wrapping div